wandb: Starting wandb agent üïµÔ∏è
2023-12-15 19:44:12,182 - wandb.wandb_agent - INFO - Running runs: []
2023-12-15 19:44:12,524 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-15 19:44:12,524 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 16
	gamma: 1.067539546755966e-06
	hidden_dim: 33
	lr: 0.0002534421638820954
	momentum: 0.0004959533702139507
	num_gc_layers: 6
	optimizer: sgd
	scheduler: CosineAnnealingLR
	step_size: 1.2128466667198838e-05
	weight_decay: 6.768061162169052e-06
2023-12-15 19:44:12,533 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=16 --gamma=1.067539546755966e-06 --hidden_dim=33 --lr=0.0002534421638820954 --momentum=0.0004959533702139507 --num_gc_layers=6 --optimizer=sgd --scheduler=CosineAnnealingLR --step_size=1.2128466667198838e-05 --weight_decay=6.768061162169052e-06
2023-12-15 19:44:17,551 - wandb.wandb_agent - INFO - Running runs: ['ssve8kk1']
INFO - 2023-12-15 19:44:21,925 - instantiator - Created a temporary directory at /tmp/tmp5hnyn4ck
INFO - 2023-12-15 19:44:21,926 - instantiator - Writing /tmp/tmp5hnyn4ck/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231215_194423-ssve8kk1
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run confused-sweep-1
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/ssve8kk1
2023-12-15 19:47:17,994 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-15 19:47:17,996 - wandb.wandb_agent - INFO - Stop: ssve8kk1
2023-12-15 19:47:23,001 - wandb.wandb_agent - INFO - Cleaning up finished run: ssve8kk1
wandb: Waiting for W&B process to finish... (failed -15). Press Control-C to abort syncing.
wandb: - 0.027 MB of 0.027 MB uploaded (0.000 MB deduped)wandb: \ 0.027 MB of 0.042 MB uploaded (0.000 MB deduped)wandb: | 0.042 MB of 0.042 MB uploaded (0.000 MB deduped)wandb: üöÄ View run confused-sweep-1 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/ssve8kk1
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231215_194423-ssve8kk1/logs
2023-12-15 19:47:29,011 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-15 19:47:29,012 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 16
	gamma: 2.642556931648941e-06
	hidden_dim: 132
	lr: 6.680805173987528e-05
	momentum: 1.1981984995291402e-06
	num_gc_layers: 2
	optimizer: sgd
	scheduler: StepLR
	step_size: 0.0007694225188661503
	weight_decay: 0.0006781083380440019
2023-12-15 19:47:29,022 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=16 --gamma=2.642556931648941e-06 --hidden_dim=132 --lr=6.680805173987528e-05 --momentum=1.1981984995291402e-06 --num_gc_layers=2 --optimizer=sgd --scheduler=StepLR --step_size=0.0007694225188661503 --weight_decay=0.0006781083380440019
2023-12-15 19:47:34,040 - wandb.wandb_agent - INFO - Running runs: ['9sz2zyhv']
INFO - 2023-12-15 19:47:36,407 - instantiator - Created a temporary directory at /tmp/tmphf2aos2e
INFO - 2023-12-15 19:47:36,407 - instantiator - Writing /tmp/tmphf2aos2e/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231215_194739-9sz2zyhv
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run mild-sweep-2
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/9sz2zyhv
2023-12-15 19:51:17,665 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-15 19:51:17,667 - wandb.wandb_agent - INFO - Stop: 9sz2zyhv
2023-12-15 19:51:22,673 - wandb.wandb_agent - INFO - Cleaning up finished run: 9sz2zyhv
2023-12-15 19:51:23,131 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-15 19:51:23,131 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 128
	gamma: 5.680323177478377e-06
	hidden_dim: 156
	lr: 0.000367626441780398
	momentum: 0.06798085599121811
	num_gc_layers: 5
	optimizer: adam
	scheduler: StepLR
	step_size: 0.0003637209658306822
	weight_decay: 0.06897872272029147
2023-12-15 19:51:23,139 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=128 --gamma=5.680323177478377e-06 --hidden_dim=156 --lr=0.000367626441780398 --momentum=0.06798085599121811 --num_gc_layers=5 --optimizer=adam --scheduler=StepLR --step_size=0.0003637209658306822 --weight_decay=0.06897872272029147
2023-12-15 19:51:28,154 - wandb.wandb_agent - INFO - Running runs: ['bcffguz6']
INFO - 2023-12-15 19:51:30,522 - instantiator - Created a temporary directory at /tmp/tmpkkz918ic
INFO - 2023-12-15 19:51:30,522 - instantiator - Writing /tmp/tmpkkz918ic/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231215_195133-bcffguz6
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run wise-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/bcffguz6
wandb: Waiting for W&B process to finish... (failed 255). Press Control-C to abort syncing.
2023-12-16 00:18:12,609 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 00:18:12,610 - wandb.wandb_agent - INFO - Stop: bcffguz6
wandb: üöÄ View run wise-sweep-3 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/bcffguz6
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231215_195133-bcffguz6/logs
2023-12-16 00:18:17,616 - wandb.wandb_agent - INFO - Cleaning up finished run: bcffguz6
2023-12-16 00:18:18,143 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 00:18:18,143 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 32
	gamma: 0.011966499453658014
	hidden_dim: 140
	lr: 0.08241453116562414
	momentum: 3.88157587526398e-06
	num_gc_layers: 1
	optimizer: sgd
	scheduler: CosineAnnealingLR
	step_size: 2.825329816407127e-05
	weight_decay: 0.006256926451801903
2023-12-16 00:18:18,156 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=32 --gamma=0.011966499453658014 --hidden_dim=140 --lr=0.08241453116562414 --momentum=3.88157587526398e-06 --num_gc_layers=1 --optimizer=sgd --scheduler=CosineAnnealingLR --step_size=2.825329816407127e-05 --weight_decay=0.006256926451801903
2023-12-16 00:18:23,179 - wandb.wandb_agent - INFO - Running runs: ['xk229x0u']
INFO - 2023-12-16 00:18:26,991 - instantiator - Created a temporary directory at /tmp/tmpqpwh5dh8
INFO - 2023-12-16 00:18:26,992 - instantiator - Writing /tmp/tmpqpwh5dh8/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_001829-xk229x0u
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run firm-sweep-2
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/xk229x0u
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run firm-sweep-2 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/xk229x0u
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_001829-xk229x0u/logs
Traceback (most recent call last):
  File "./infograph_w_discriminator/unsupervised/main.py", line 217, in <module>
    loss.backward()
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/_tensor.py", line 488, in backward
    torch.autograd.backward(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/autograd/__init__.py", line 197, in backward
    Variable._execution_engine.run_backward(  # Calls into the C++ engine to run the backward pass
RuntimeError: Trying to backward through the graph a second time (or directly access saved tensors after they have already been freed). Saved intermediate values of the graph are freed when you call .backward() or autograd.grad(). Specify retain_graph=True if you need to backward through the graph a second time or if you need to access saved tensors after calling backward.
2023-12-16 01:39:46,932 - wandb.wandb_agent - INFO - Cleaning up finished run: xk229x0u
2023-12-16 01:39:47,317 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 01:39:47,317 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 64
	gamma: 1.1446631947296546e-06
	hidden_dim: 173
	lr: 1.0749818077291928e-05
	momentum: 0.04357723329516462
	num_gc_layers: 5
	optimizer: adam
	scheduler: StepLR
	step_size: 0.07818659322023308
	weight_decay: 0.01716947031054622
2023-12-16 01:39:47,329 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=64 --gamma=1.1446631947296546e-06 --hidden_dim=173 --lr=1.0749818077291928e-05 --momentum=0.04357723329516462 --num_gc_layers=5 --optimizer=adam --scheduler=StepLR --step_size=0.07818659322023308 --weight_decay=0.01716947031054622
2023-12-16 01:39:52,346 - wandb.wandb_agent - INFO - Running runs: ['pn2o8cwi']
INFO - 2023-12-16 01:39:56,454 - instantiator - Created a temporary directory at /tmp/tmpmrq6gbw8
INFO - 2023-12-16 01:39:56,454 - instantiator - Writing /tmp/tmpmrq6gbw8/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_013959-pn2o8cwi
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run resilient-sweep-2
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/pn2o8cwi
2023-12-16 01:43:59,805 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 01:43:59,808 - wandb.wandb_agent - INFO - Stop: pn2o8cwi
2023-12-16 01:44:04,814 - wandb.wandb_agent - INFO - Cleaning up finished run: pn2o8cwi
2023-12-16 01:44:05,175 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 01:44:05,175 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 64
	gamma: 0.0011376100333456392
	hidden_dim: 212
	lr: 0.0662915457865508
	momentum: 0.03737203986070635
	num_gc_layers: 6
	optimizer: adam
	scheduler: StepLR
	step_size: 0.041210609892738094
	weight_decay: 0.00190144299568342
2023-12-16 01:44:05,194 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=64 --gamma=0.0011376100333456392 --hidden_dim=212 --lr=0.0662915457865508 --momentum=0.03737203986070635 --num_gc_layers=6 --optimizer=adam --scheduler=StepLR --step_size=0.041210609892738094 --weight_decay=0.00190144299568342
2023-12-16 01:44:10,213 - wandb.wandb_agent - INFO - Running runs: ['p20jufso']
INFO - 2023-12-16 01:44:14,631 - instantiator - Created a temporary directory at /tmp/tmp972cnbic
INFO - 2023-12-16 01:44:14,632 - instantiator - Writing /tmp/tmp972cnbic/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_014417-p20jufso
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run comfy-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/p20jufso
2023-12-16 01:45:48,334 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 01:45:48,337 - wandb.wandb_agent - INFO - Stop: p20jufso
2023-12-16 01:45:53,342 - wandb.wandb_agent - INFO - Cleaning up finished run: p20jufso
2023-12-16 01:45:53,790 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 01:45:53,790 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 64
	gamma: 5.582729072073819e-06
	hidden_dim: 39
	lr: 0.0005635385101331201
	momentum: 0.00029243408569222697
	num_gc_layers: 4
	optimizer: adam
	scheduler: StepLR
	step_size: 0.0006218309339598731
	weight_decay: 0.0008495312304076916
2023-12-16 01:45:53,803 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=64 --gamma=5.582729072073819e-06 --hidden_dim=39 --lr=0.0005635385101331201 --momentum=0.00029243408569222697 --num_gc_layers=4 --optimizer=adam --scheduler=StepLR --step_size=0.0006218309339598731 --weight_decay=0.0008495312304076916
2023-12-16 01:45:58,821 - wandb.wandb_agent - INFO - Running runs: ['vogxhxiw']
INFO - 2023-12-16 01:46:03,165 - instantiator - Created a temporary directory at /tmp/tmpg2zfdc00
INFO - 2023-12-16 01:46:03,166 - instantiator - Writing /tmp/tmpg2zfdc00/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_014605-vogxhxiw
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run deft-sweep-4
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/vogxhxiw
wandb: Waiting for W&B process to finish... (failed 255). Press Control-C to abort syncing.
2023-12-16 01:47:57,467 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 01:47:57,468 - wandb.wandb_agent - INFO - Stop: vogxhxiw
wandb: üöÄ View run deft-sweep-4 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/vogxhxiw
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_014605-vogxhxiw/logs
2023-12-16 01:48:02,473 - wandb.wandb_agent - INFO - Cleaning up finished run: vogxhxiw
2023-12-16 01:48:02,968 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 01:48:02,968 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 128
	gamma: 0.0001232958621256232
	hidden_dim: 57
	lr: 0.0007421024284213682
	momentum: 1.9889841955879452e-05
	num_gc_layers: 10
	optimizer: adam
	scheduler: CosineAnnealingLR
	step_size: 0.0002115536797182813
	weight_decay: 0.0860498086911267
2023-12-16 01:48:02,978 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=128 --gamma=0.0001232958621256232 --hidden_dim=57 --lr=0.0007421024284213682 --momentum=1.9889841955879452e-05 --num_gc_layers=10 --optimizer=adam --scheduler=CosineAnnealingLR --step_size=0.0002115536797182813 --weight_decay=0.0860498086911267
2023-12-16 01:48:07,993 - wandb.wandb_agent - INFO - Running runs: ['teofqk6z']
INFO - 2023-12-16 01:48:11,473 - instantiator - Created a temporary directory at /tmp/tmpemmehkfj
INFO - 2023-12-16 01:48:11,474 - instantiator - Writing /tmp/tmpemmehkfj/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_014814-teofqk6z
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run stellar-sweep-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/teofqk6z
wandb: Waiting for W&B process to finish... (failed 255). Press Control-C to abort syncing.
2023-12-16 01:49:04,848 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 01:49:04,848 - wandb.wandb_agent - INFO - Stop: teofqk6z
wandb: üöÄ View run stellar-sweep-5 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/teofqk6z
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_014814-teofqk6z/logs
2023-12-16 01:49:09,853 - wandb.wandb_agent - INFO - Cleaning up finished run: teofqk6z
2023-12-16 01:49:10,368 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 01:49:10,368 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 32
	gamma: 1.2484714239337742e-05
	hidden_dim: 206
	lr: 0.008182301317061784
	momentum: 0.0001429075973581153
	num_gc_layers: 2
	optimizer: sgd
	scheduler: CosineAnnealingLR
	step_size: 0.06471218763883324
	weight_decay: 4.368983535307882e-06
2023-12-16 01:49:10,381 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=32 --gamma=1.2484714239337742e-05 --hidden_dim=206 --lr=0.008182301317061784 --momentum=0.0001429075973581153 --num_gc_layers=2 --optimizer=sgd --scheduler=CosineAnnealingLR --step_size=0.06471218763883324 --weight_decay=4.368983535307882e-06
2023-12-16 01:49:15,396 - wandb.wandb_agent - INFO - Running runs: ['est65bq6']
INFO - 2023-12-16 01:49:18,359 - instantiator - Created a temporary directory at /tmp/tmpd8i97gpo
INFO - 2023-12-16 01:49:18,359 - instantiator - Writing /tmp/tmpd8i97gpo/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_014921-est65bq6
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run smart-sweep-6
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/est65bq6
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run smart-sweep-6 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/est65bq6
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_014921-est65bq6/logs
Traceback (most recent call last):
  File "./infograph_w_discriminator/unsupervised/main.py", line 206, in <module>
    loss = model(data.x, data.y, data.edge_index, data.batch, data.num_graphs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "./infograph_w_discriminator/unsupervised/main.py", line 119, in forward
    mlp_outputs = self.mlp_model(x)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "./infograph_w_discriminator/unsupervised/main.py", line 48, in forward
    return self.layers(x)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/container.py", line 204, in forward
    input = module(input)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: mat1 and mat2 shapes cannot be multiplied (34830x3 and 412x206)
2023-12-16 05:12:51,659 - wandb.wandb_agent - INFO - Cleaning up finished run: est65bq6
2023-12-16 05:12:52,173 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 05:12:52,173 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 16
	gamma: 1.6931699473398191e-06
	hidden_dim: 54
	lr: 0.015376948045815847
	momentum: 0.00011288681751305076
	num_gc_layers: 10
	optimizer: sgd
	scheduler: CosineAnnealingLR
	step_size: 1.5889229784382212e-05
	weight_decay: 4.1972999150676e-06
2023-12-16 05:12:52,196 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=16 --gamma=1.6931699473398191e-06 --hidden_dim=54 --lr=0.015376948045815847 --momentum=0.00011288681751305076 --num_gc_layers=10 --optimizer=sgd --scheduler=CosineAnnealingLR --step_size=1.5889229784382212e-05 --weight_decay=4.1972999150676e-06
2023-12-16 05:12:57,218 - wandb.wandb_agent - INFO - Running runs: ['204d15jj']
INFO - 2023-12-16 05:13:01,361 - instantiator - Created a temporary directory at /tmp/tmp4b4qzb6j
INFO - 2023-12-16 05:13:01,362 - instantiator - Writing /tmp/tmp4b4qzb6j/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_051304-204d15jj
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run lunar-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/204d15jj
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: üöÄ View run lunar-sweep-3 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/204d15jj
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_051304-204d15jj/logs
Traceback (most recent call last):
  File "./infograph_w_discriminator/unsupervised/main.py", line 206, in <module>
    loss = model(data.x, data.y, data.edge_index, data.batch, data.num_graphs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "./infograph_w_discriminator/unsupervised/main.py", line 119, in forward
    mlp_outputs = self.mlp_model(x)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "./infograph_w_discriminator/unsupervised/main.py", line 48, in forward
    return self.layers(x)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/container.py", line 204, in forward
    input = module(input)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/module.py", line 1194, in _call_impl
    return forward_call(*input, **kwargs)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/torch/nn/modules/linear.py", line 114, in forward
    return F.linear(input, self.weight, self.bias)
RuntimeError: mat1 and mat2 shapes cannot be multiplied (17369x3 and 540x270)
2023-12-16 09:16:41,757 - wandb.wandb_agent - INFO - Cleaning up finished run: 204d15jj
2023-12-16 09:16:42,192 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 09:16:42,192 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 128
	gamma: 5.2799889004080346e-05
	hidden_dim: 225
	lr: 1.489537073990187e-06
	momentum: 0.00045583530430019813
	num_gc_layers: 5
	optimizer: adam
	scheduler: StepLR
	step_size: 7.505983457223794e-05
	weight_decay: 1.150226132136168e-05
2023-12-16 09:16:42,205 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=128 --gamma=5.2799889004080346e-05 --hidden_dim=225 --lr=1.489537073990187e-06 --momentum=0.00045583530430019813 --num_gc_layers=5 --optimizer=adam --scheduler=StepLR --step_size=7.505983457223794e-05 --weight_decay=1.150226132136168e-05
2023-12-16 09:16:47,225 - wandb.wandb_agent - INFO - Running runs: ['ngawu2t9']
INFO - 2023-12-16 09:16:50,795 - instantiator - Created a temporary directory at /tmp/tmppo8bo8rl
INFO - 2023-12-16 09:16:50,795 - instantiator - Writing /tmp/tmppo8bo8rl/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_091653-ngawu2t9
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run earnest-sweep-4
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/ngawu2t9
2023-12-16 10:26:49,904 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 10:26:49,905 - wandb.wandb_agent - INFO - Stop: ngawu2t9
2023-12-16 10:26:54,910 - wandb.wandb_agent - INFO - Cleaning up finished run: ngawu2t9
2023-12-16 10:26:55,392 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 10:26:55,392 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 128
	gamma: 0.02289513973810553
	hidden_dim: 255
	lr: 0.00018893534919460153
	momentum: 0.00013480007390376028
	num_gc_layers: 1
	optimizer: sgd
	scheduler: StepLR
	step_size: 0.012734597616451128
	weight_decay: 0.0010664831414188925
2023-12-16 10:26:55,401 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=128 --gamma=0.02289513973810553 --hidden_dim=255 --lr=0.00018893534919460153 --momentum=0.00013480007390376028 --num_gc_layers=1 --optimizer=sgd --scheduler=StepLR --step_size=0.012734597616451128 --weight_decay=0.0010664831414188925
2023-12-16 10:27:00,417 - wandb.wandb_agent - INFO - Running runs: ['pxhw3k6t']
INFO - 2023-12-16 10:27:02,687 - instantiator - Created a temporary directory at /tmp/tmpdk2hzmld
INFO - 2023-12-16 10:27:02,687 - instantiator - Writing /tmp/tmpdk2hzmld/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_102705-pxhw3k6t
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run fine-sweep-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/pxhw3k6t
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: 
wandb: Run history:
wandb: epoch ‚ñÅ‚ñÉ‚ñÖ‚ñÜ‚ñà
wandb: 
wandb: Run summary:
wandb:  Loss nan
wandb: epoch 5
wandb: 
wandb: üöÄ View run fine-sweep-5 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/pxhw3k6t
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_102705-pxhw3k6t/logs
Traceback (most recent call last):
  File "./infograph_w_discriminator/unsupervised/main.py", line 219, in <module>
    res = evaluate_embedding(emb, y, classification)
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 193, in evaluate_embedding
    randomforest_accuracies = [randomforest_regressor(x, y, search) for _ in range(1)]
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 193, in <listcomp>
    randomforest_accuracies = [randomforest_regressor(x, y, search) for _ in range(1)]
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 145, in randomforest_regressor
    regressor.fit(x_train, y_train)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 875, in fit
    self._run_search(evaluate_candidates)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 1375, in _run_search
    evaluate_candidates(ParameterGrid(self.param_grid))
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 852, in evaluate_candidates
    _warn_or_raise_about_fit_failures(out, self.error_score)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 367, in _warn_or_raise_about_fit_failures
    raise ValueError(all_fits_failed_message)
ValueError: 
All the 12 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
12 fits failed with the following error:
Traceback (most recent call last):
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 686, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/ensemble/_forest.py", line 331, in fit
    X, y = self._validate_data(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/base.py", line 596, in _validate_data
    X, y = check_X_y(X, y, **check_params)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 1074, in check_X_y
    X = check_array(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 899, in check_array
    _assert_all_finite(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 146, in _assert_all_finite
    raise ValueError(msg_err)
ValueError: Input X contains NaN.
RandomForestRegressor does not accept missing values encoded as NaN natively. For supervised learning, you might want to consider sklearn.ensemble.HistGradientBoostingClassifier and Regressor which accept missing values encoded as NaNs natively. Alternatively, it is possible to preprocess the data, for instance by using an imputer transformer in a pipeline or drop samples with missing values. See https://scikit-learn.org/stable/modules/impute.html You can find a list of all estimators that handle NaN values at the following page: https://scikit-learn.org/stable/modules/impute.html#estimators-that-handle-nan-values

2023-12-16 13:52:12,374 - wandb.wandb_agent - INFO - Cleaning up finished run: pxhw3k6t
2023-12-16 13:52:12,942 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 13:52:12,942 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 32
	gamma: 2.6268258223955596e-06
	hidden_dim: 185
	lr: 0.001246341676242215
	momentum: 0.09223195431822456
	num_gc_layers: 6
	optimizer: sgd
	scheduler: StepLR
	step_size: 0.021087634215162045
	weight_decay: 2.4845470341568658e-05
2023-12-16 13:52:12,953 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=32 --gamma=2.6268258223955596e-06 --hidden_dim=185 --lr=0.001246341676242215 --momentum=0.09223195431822456 --num_gc_layers=6 --optimizer=sgd --scheduler=StepLR --step_size=0.021087634215162045 --weight_decay=2.4845470341568658e-05
2023-12-16 13:52:17,970 - wandb.wandb_agent - INFO - Running runs: ['2rw3jra2']
INFO - 2023-12-16 13:52:20,112 - instantiator - Created a temporary directory at /tmp/tmp3cwpj3bb
INFO - 2023-12-16 13:52:20,112 - instantiator - Writing /tmp/tmp3cwpj3bb/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_135222-2rw3jra2
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run faithful-sweep-3
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/2rw3jra2
wandb: Waiting for W&B process to finish... (failed 1). Press Control-C to abort syncing.
wandb: 
wandb: Run history:
wandb: epoch ‚ñÅ‚ñÉ‚ñÖ‚ñÜ‚ñà
wandb: 
wandb: Run summary:
wandb:  Loss nan
wandb: epoch 5
wandb: 
wandb: üöÄ View run faithful-sweep-3 at: https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/2rw3jra2
wandb: Synced 6 W&B file(s), 0 media file(s), 0 artifact file(s) and 0 other file(s)
wandb: Find logs at: ./wandb/run-20231216_135222-2rw3jra2/logs
Traceback (most recent call last):
  File "./infograph_w_discriminator/unsupervised/main.py", line 219, in <module>
    res = evaluate_embedding(emb, y, classification)
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 193, in evaluate_embedding
    randomforest_accuracies = [randomforest_regressor(x, y, search) for _ in range(1)]
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 193, in <listcomp>
    randomforest_accuracies = [randomforest_regressor(x, y, search) for _ in range(1)]
  File "/u/home/wyo/digital_twin/scripts/infograph_w_discriminator/unsupervised/evaluate_embedding.py", line 145, in randomforest_regressor
    regressor.fit(x_train, y_train)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 875, in fit
    self._run_search(evaluate_candidates)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 1375, in _run_search
    evaluate_candidates(ParameterGrid(self.param_grid))
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_search.py", line 852, in evaluate_candidates
    _warn_or_raise_about_fit_failures(out, self.error_score)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 367, in _warn_or_raise_about_fit_failures
    raise ValueError(all_fits_failed_message)
ValueError: 
All the 12 fits failed.
It is very likely that your model is misconfigured.
You can try to debug the error by setting error_score='raise'.

Below are more details about the failures:
--------------------------------------------------------------------------------
12 fits failed with the following error:
Traceback (most recent call last):
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/model_selection/_validation.py", line 686, in _fit_and_score
    estimator.fit(X_train, y_train, **fit_params)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/ensemble/_forest.py", line 331, in fit
    X, y = self._validate_data(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/base.py", line 596, in _validate_data
    X, y = check_X_y(X, y, **check_params)
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 1074, in check_X_y
    X = check_array(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 899, in check_array
    _assert_all_finite(
  File "/u/home/wyo/.conda/envs/digital_twin/lib/python3.8/site-packages/sklearn/utils/validation.py", line 146, in _assert_all_finite
    raise ValueError(msg_err)
ValueError: Input X contains NaN.
RandomForestRegressor does not accept missing values encoded as NaN natively. For supervised learning, you might want to consider sklearn.ensemble.HistGradientBoostingClassifier and Regressor which accept missing values encoded as NaNs natively. Alternatively, it is possible to preprocess the data, for instance by using an imputer transformer in a pipeline or drop samples with missing values. See https://scikit-learn.org/stable/modules/impute.html You can find a list of all estimators that handle NaN values at the following page: https://scikit-learn.org/stable/modules/impute.html#estimators-that-handle-nan-values

2023-12-16 22:39:16,334 - wandb.wandb_agent - INFO - Cleaning up finished run: 2rw3jra2
2023-12-16 22:39:16,930 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 22:39:16,930 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 16
	gamma: 2.18393361261386e-06
	hidden_dim: 182
	lr: 0.0007635982242431687
	momentum: 0.000479109966979774
	num_gc_layers: 9
	optimizer: adam
	scheduler: CosineAnnealingLR
	step_size: 3.004689067713789e-05
	weight_decay: 0.01476544248562428
2023-12-16 22:39:16,946 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=16 --gamma=2.18393361261386e-06 --hidden_dim=182 --lr=0.0007635982242431687 --momentum=0.000479109966979774 --num_gc_layers=9 --optimizer=adam --scheduler=CosineAnnealingLR --step_size=3.004689067713789e-05 --weight_decay=0.01476544248562428
2023-12-16 22:39:21,961 - wandb.wandb_agent - INFO - Running runs: ['xthtx46e']
INFO - 2023-12-16 22:39:25,766 - instantiator - Created a temporary directory at /tmp/tmpmkkk9y7d
INFO - 2023-12-16 22:39:25,766 - instantiator - Writing /tmp/tmpmkkk9y7d/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_223928-xthtx46e
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run desert-sweep-4
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/xthtx46e
2023-12-16 22:57:43,012 - wandb.wandb_agent - INFO - Agent received command: stop
2023-12-16 22:57:43,013 - wandb.wandb_agent - INFO - Stop: xthtx46e
2023-12-16 22:57:48,018 - wandb.wandb_agent - INFO - Cleaning up finished run: xthtx46e
2023-12-16 22:57:48,537 - wandb.wandb_agent - INFO - Agent received command: run
2023-12-16 22:57:48,537 - wandb.wandb_agent - INFO - Agent starting run with config:
	batchs: 64
	gamma: 0.00042716378730608143
	hidden_dim: 199
	lr: 1.0287211483944313e-05
	momentum: 0.03643043878911333
	num_gc_layers: 3
	optimizer: adam
	scheduler: StepLR
	step_size: 0.0010234880742267668
	weight_decay: 2.374081895366951e-06
2023-12-16 22:57:48,545 - wandb.wandb_agent - INFO - About to run command: /usr/bin/env python ./infograph_w_discriminator/unsupervised/main.py --batchs=64 --gamma=0.00042716378730608143 --hidden_dim=199 --lr=1.0287211483944313e-05 --momentum=0.03643043878911333 --num_gc_layers=3 --optimizer=adam --scheduler=StepLR --step_size=0.0010234880742267668 --weight_decay=2.374081895366951e-06
2023-12-16 22:57:53,561 - wandb.wandb_agent - INFO - Running runs: ['jgm3b0qr']
INFO - 2023-12-16 22:57:56,386 - instantiator - Created a temporary directory at /tmp/tmpr_0n5p3w
INFO - 2023-12-16 22:57:56,386 - instantiator - Writing /tmp/tmpr_0n5p3w/_remote_module_non_scriptable.py
wandb: Currently logged in as: yussufwaly. Use `wandb login --relogin` to force relogin
wandb: WARNING Ignored wandb.init() arg project when running a sweep.
wandb: WARNING Ignored wandb.init() arg entity when running a sweep.
wandb: wandb version 0.16.1 is available!  To upgrade, please run:
wandb:  $ pip install wandb --upgrade
wandb: Tracking run with wandb version 0.15.3
wandb: Run data is saved locally in /u/home/wyo/digital_twin/scripts/wandb/run-20231216_225759-jgm3b0qr
wandb: Run `wandb offline` to turn off syncing.
wandb: Syncing run sunny-sweep-5
wandb: ‚≠êÔ∏è View project at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator
wandb: üßπ View sweep at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/sweeps/4v305ag8
wandb: üöÄ View run at https://wandb.ai/yussufwaly/digital_twin_infograph_w_discriminator/runs/jgm3b0qr
